.github/CODEOWNERS

```diff
@@ -4,15 +4,21 @@
 accounts/usbwallet              @karalabe
 accounts/scwallet               @gballet
 accounts/abi                    @gballet
+cmd/clef                        @holiman
+cmd/puppeth                     @karalabe
 consensus                       @karalabe
-core/                           @karalabe @holiman
-eth/                            @karalabe
+consensus/clique                @holiman @karalabe
```

Comment:
Not sure I follow the addition of Martin here.

Reply:
My misunderstanding. Removed.